src/scanner.jl

```diff
@@ -1419,6 +1474,9 @@ function scan_plain_spaces(stream::TokenStream, indent::Integer,
                 forwardchars!(stream)
             else
                 push!(breaks, scan_line_break(stream))
+                if peek(stream.input) == '\uFEFF'
```

Comment:
Can you add something to test this branch?

Reply:
Yeah, that's a good idea. IIRC it's triggered for a YAML document with just a bare string:
```
---
just a string
\uFEFF---
another document
```